### ISSUE TITLE
Git menu will trigger commit call on every click

### ISSUE THREAD

**Issue by ewoutkramer** [microsoft/FHIR-Converter#86]
**Describe the bug**
When there are changes to commit, every single click on the Git dropdown menu will start a commit API call.

**To Reproduce**
1. Make some changes to the handlebar templates
2. Click on the "Git" menu
3. Watch the developer console to see the app invoke /commit on every click on the menu

**Expected behavior**
Only invoke this call when the "Commit changes" item is called

The callback for the commit is attached to the menu, instead of the "Commit changes" item.

**Comment by ewoutkramer** [comment]
A PR will follow shortly.

**Comment by ACMoretxj** [comment]
Hi Ewout, I can't reproduce the issue, it seems that the dropdown menu and the commit link(inside the dropdown menu) all work normally without functional interleaving on my machine. Could you please provide more detailed context information or screenshots?

**Comment by ewoutkramer** [comment]
The event that triggers the call to commit seems to be attached to the element `#git-dropdown` - when I had commitable changes, every time I would just click the dropdown so it would open, I saw a commit being sent to the api endpoint.  So, I changed to code to be attached to the `<a>` link.  But I see now that the `on()` actually has the name of the child anchor there.  Hum.  

And I cannot reproduce it anymore, since I upgraded my node installation, and now all the Git functions don't work anymore anyway.  I'll try downgrading node and see if I can reproduce it again.
